src/functionalTest/resources/features/F-018/F-018.feature

```diff
@@ -0,0 +1,123 @@
+    And a successful call [to delete role assignments just created above] as in [S-266_DeleteDataForRoleAssignmentsForOrgRoles].
+
+  @S-267
+  Scenario: must successfully create tribunal-member-2 ET_EnglandWales case role
+    Given a user with [an active IDAM profile with full permissions],
+    And a user [Befta3 - who is the actor for requested role],
+    And a successful call [to create org role assignments for actors & requester] as in [S-267_Org_Role_Creation],
+    When a request is prepared with appropriate values,
+    And the request [contains tribunal-member-2 case role assignment],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a positive response is received,
+    And the response has all other details as expected,
+    And a successful call [to delete case role assignment for the same actor] as in [S-267_Delete_Case_Role],
+    And a successful call [to delete role assignments just created above] as in [S-267_DeleteDataForRoleAssignmentsForOrgRoles].
+
+  @S-268
+  Scenario: must successfully create challenged-access-legal-ops role
+    Given a user with [an active IDAM profile with full permissions],
+    And a user [Befta1 - who is the actor for requested role],
+    And a successful call [to create org role assignments for actors & requester] as in [S-268_Org_Role_Creation],
+    When a request is prepared with appropriate values,
+    And the request [contains challenged-access-legal-ops case requested role assignment],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a positive response is received,
+    And the response has all other details as expected.
+    And a successful call [to delete role assignments just created above] as in [S-268_DeleteDataForRoleAssignmentsForOrgRoles],
+    And a successful call [to delete role assignments just created above] as in [S-268_DeleteDataForRoleAssignmentsForChallengedAccess].
+
+  @S-269
+  Scenario: must successfully create specific access granted role for ET judiciary
+    Given a user with [an active IDAM profile with full permissions],
+    And a user [Befta1 - who is the actor for requested role],
+    And a successful call [to create org role assignments for actors & requester] as in [S-269_Org_Role_Creation],
+    And a successful call [to create role assignments for requested role] as in [S-269_Access_Requested],
+    When a request is prepared with appropriate values,
+    And the request [contains specific-access-judiciary case granted role assignment],
+    And it is submitted to call the [Create Role Assignments] operation of [Role Assignments Service],
+    Then a positive response is received,
+    And the response has all other details as expected.
+    And a successful call [to delete role assignments just created above] as in [S-269_DeleteDataForRoleAssignmentsForOrgRoles].
```

Comment:
Are we missing the second delete from here: i.e. `S-269_DeleteDataForRoleAssignmentsForGrantedRole`?

Reply:
Removed the S-269 test as we are not able to remove the specific access case until it's expiry.